app/domain/authentication/authn_gcp/decoded_token.rb

```diff
@@ -50,6 +50,7 @@ def optional_token_claim_value(optional_token_claim)
         optional_token_claim_value = token_claim_value(optional_token_claim)
 
         if optional_token_claim_value.nil? || optional_token_claim_value.empty?
+          optional_token_claim_value = nil
```

Comment:
Why does it have to be `nil`?

Reply:
i prefer to have 1  value which indicate not exists, so i choose between nil and empty i choose nil